deploy/charts/external-secrets/values.yaml

```diff
@@ -194,6 +194,13 @@ serviceMonitor:
   # -- Specifies whether to create a ServiceMonitor resource for collecting Prometheus metrics
   enabled: false
 
+  # -- How should we react to missing CRD "monitoring.coreos.com/v1"
+  # Possible values:
+  # skipIfMissing - Only render ServiceMonitor resources if CRD is present, skip if missing
+  # failIfMissing - Fail Helm install if CRD is not present
+  # alwaysRender  - Always render ServiceMonitor resources, do not check for CRD
```

Comment:
```suggestion
  # @schema
  # enum:
  # - skipIfMissing
  # - failIfMissing
  # - alwaysRender
  # @schema
  # -- How should we react to missing CRD "monitoring.coreos.com/v1"
  # Possible values:
  # skipIfMissing - Only render ServiceMonitor resources if CRD is present, skip if missing
  # failIfMissing - Fail Helm install if CRD is not present
  # alwaysRender  - Always render ServiceMonitor resources, do not check for CRD
```

Reply:
This will update the helm schema to not be a string, and instead be an enum (ref: https://github.com/dadav/helm-schema?tab=readme-ov-file#enum).